packages/accounts/src/index.ts

```diff
@@ -1,2 +1,11 @@
 // Add you exports here, make sure to export types separately from impls and use the `type` keyword when exporting them
 // Don't use wildcard exports, instead use named exports
+
+//kernel exports
+export { KernelFactoryAbi } from "./kernel-zerodev/abis/KernelFactoryAbi";
+export { KernelAccountAbi } from "./kernel-zerodev/abis/KernelAccountAbi";
+export type { KernelSmartAccountParams } from "./kernel-zerodev/account";
+export { KernelSmartContractAccount } from "./kernel-zerodev/account";
+export { KernelAccountProvider } from "./kernel-zerodev/provider";
+export { BaseValidator } from "./kernel-zerodev/validator/base";
+export { ECDSAValidator } from "./kernel-zerodev/validator/ecdsa";
```

Comment:
one nit on these names, maybe we should also prefix these with `Kernel*` as well. That way if some other account also has a concept of validators they won't clash

Reply:
Ok